volume_data_converter/create_volume_viewer_data.py

```diff
@@ -6,14 +6,27 @@
 from volume_data_converter import scoord_du_new2 as scoor_du
 from typing import List
 
+import json
+import sys
+from pathlib import Path
 
 app = typer.Typer()
 
+constants_file_path = os.path.join(
+    Path(__file__).absolute().parent, "config", "constants.json"
+)
+
 
 def fprintf(stream, format_spec, *args):
     stream.write(format_spec % args)
 
 
+def getVariableValOrDefault(configuration_file, nc_file_handler, variable_name: str):
```

Comment:
I deleted the function. I dodnt realize there is already a function in python that query's data from a dictionary  and sets up a default value in case it doesnt find it.